config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -19,14 +19,14 @@ periodics:
       - --provider=gce
       - --gcp-region=us-central1
       - --gcp-node-image=gci
-      - --extract=ci/latest-1.32
+      - --extract=ci/latest-1.33
       - --timeout=180m
       - --env=KUBE_PROXY_DAEMONSET=true
       - --env=ENABLE_POD_PRIORITY=true
       - --env=KUBE_FEATURE_GATES=AllAlpha=true,EventedPLEG=false
       - --env=ENABLE_CACHE_MUTATION_DETECTOR=true
       - --runtime-config=api/all=true
-      - --test_args=--ginkgo.focus=\[Feature:(AdmissionWebhookMatchConditions|InPlacePodVerticalScaling|SidecarContainers|StorageVersionAPI|PodPreset|StatefulSetAutoDeletePVC|ClusterTrustBundle|ClusterTrustBundleProjection)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance|Example)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
```

Comment:
This looks a little sketchy. Can we check where `ClusterTrustBundle|ClusterTrustBundleProjection` came from and eventually update the jobs config (`test_config.yaml`)?

Reply:
Any tips on how to back trace this? Would you consider it blocking for merge?

Reply:
I would consider it a blocker because we would be running different tests than before and I'm almost sure we don't want that.

Reply:
I think this change is expected based on https://github.com/kubernetes/test-infra/commit/b3af75ec14ce5016af18010d120cbaaf1b536d97

I don't know why these jobs didn't get updated earlier, but looks like those arguments shouldn't be present indeed, so we're all good here.